lib/trento/application/projectors/database_projector.ex

```diff
@@ -302,11 +302,16 @@ defmodule Trento.DatabaseProjector do
     database =
       DatabaseReadModel
       |> Repo.get!(sap_system_id)
-      |> Repo.preload([:tags])
+      |> Repo.preload([
+        :tags,
+        :database_instances,
+        database_instances: [host: :tags],
```

Comment:
Why do we need this `host and tags` preloaded instances?
Why do we even need to have them twice?

Reply:
I just included them otherwise the JSON serialiser was complaining.

>Why do we even need to have them twice?

One is the `:tags` for the `DatabaseReadModel`, the other is `:tags` for `HostReadModel`.

I can see how to remove the `HostReadModel` tags if it's not desired.

Reply:
So we need to fix the view them, not add `database_instances` twice loading hosts that we don't need, right?

Reply:
So you don't want to include the hosts data from `database_instances`?

Reply:
Is there any reason why we need it?
We don't use it in the frontend as far as I know